packages/calypso-babel-config/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+	"extends": "@automattic/calypso-typescript-config/js-package.json",
+	"include": [ "config.js", "presets/*.js" ]
```

Comment:
why do we need a tsconfig if we aren't using TS?

Reply:
We have `tsconfig.json` in all packages, both JS and TS. The rationale is that it helps editors like VSCode (or more broadly, it helps IDEs that use https://github.com/Microsoft/TypeScript/wiki/Standalone-Server-%28tsserver%29 for language services).

Reply:
Cool!